sdks/python/apache_beam/examples/complete/autocomplete.py

```diff
@@ -53,7 +53,7 @@ def run(argv=None):
    | 'format' >> beam.Map(
        lambda (prefix, candidates): '%s: %s' % (prefix, candidates))
    | 'write' >> WriteToText(known_args.output))
-  p.run()
+  p.run().wait_until_finish()
```

Comment:
Do we still need this? What happens if we don't have it? (I'd assume these are non-daemon threads we're starting, so the pipeline will still continue to run until complete before the process exits, right?